bitcoin/tx.c

```diff
@@ -180,7 +180,36 @@ static int elements_tx_add_fee_output(struct bitcoin_tx *tx)
 void bitcoin_tx_set_locktime(struct bitcoin_tx *tx, u32 locktime)
 {
 	tx->wtx->locktime = locktime;
-	tx->psbt->tx->locktime = locktime;
+	tx->psbt->fallback_locktime = locktime;
+	tx->psbt->has_fallback_locktime = true;
+}
+
+/* FIXME Stolen from psbt_append_input; export? */
+static struct wally_tx_input *wally_tx_input_from_outpoint_sequence(const struct bitcoin_outpoint *outpoint,
+			u32 sequence)
+{
+	struct wally_tx_input *tx_in;
```

Comment:
```suggestion
	struct wally_tx_input *tx_in;
```
here there is a space nit